For Namada v101.0.0 (#4342)
informalsystems/hermes: .changelog/unreleased/breaking-changes/4342-namada-v101.md

> - Update for Namada v101.0.0. This changes don't work with Namada v1.1.x.

Reviewer: ```suggestion
- Update for Namada v101.0.0. This update is not compatible with Namada v1.1.x.
```
